README.md

```diff
@@ -107,7 +129,7 @@ generator = outlines.generate.choice(model, Sentiment)
 answer = generator(prompt)
 ````
```

Comment:
Seems this part also needs to be adjusted, but maybe we can show only the difference:

````python
from enum import Enum

class Food(str, Enum):
      pizza = "Pizza"
      pasta = "Pasta"
      salad = "Salad"
      dessert = "Dessert"

...

generator = outlines.generate.choice(model, Food)
````

Reply:
IMO every example in a README should be completely copy-able with no slice-and-dice on the user's part. This is of course personal preference, so up to y'all.

Reply:
If one day we generate documentation with `mdbook`, it offers a feature for [hiding code lines](https://rust-lang.github.io/mdBook/format/mdbook.html#hiding-code-lines) from the user. I realized there isn't a similar feature in GitHub Flavored Markdown _(yet?)_. The closest thing I'm aware of is [collapsed sections](https://github.com/github/docs/blob/main/content/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections.md)...